docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "2"
+services:
+  node:
+    image: "node:14"
+    working_dir: /usr/opis-manager
+    ports:
+      - "4200:4200"
+    volumes:
+      - ./:/usr/opis-manager
+    networks:
+      - opis
+    command: bash -c "npm run dev"
+
+networks:
+  opis:
+    external: true
```

Comment:
Perché utilizzare una rete esterna? 
Inoltre qualora si volesse utilizzare una rete esterna è necessario passare il parametro `name` per indicare la suddetta rete.

Reply:
In realtà avevo creato la rete solo per sviluppo in locale, mi serviva far comunicare il container delle api con questo e per "organizzare" un po' meglio avevo pensato di utilizzare una user-defined network. 
Effettivamente però mi sa di aver creato separatamente la rete durante i test  (direttamente da riga di comando) e di aver dimenticato di aggiornare poi il docker-compose specificando la creazione della stessa😅

Reply:
Io utilizzerei la rete di default, anche perché in realtà poi utilizzeremo, al solito, nginx come reverse proxy. Direi di inserire il docker- compose in uno dei due progetti e lascerei tutto nella rete di default.
Non dimenticare però di documentare anche le porte esposte dal servizio che saranno sempre utili a chi lavora al progetto!
Inoltre bisognerebbe fare l'action di GitHub per la creazione automatica dell'immagine docker ma in caso questa posso crearla anche io :)
Grazie mille!

Reply:
Perfetto! (Scusa per il ritardo 😅)
Allora pensavo di pushare il compose delle API su docker hub, in modo tale da aggiungere a questo docker-compose il servizio api facendo pullare direttamente dall'hub; in questo modo chiunque debba lavorare al progetto potrebbe far partire tutto tramite un semplice compose up.
Cosa ne pensi?

Reply:
Docker Hub purtroppo non permette più la build sui loro server. Per questa ragione, per gli altri progetti, stiamo utilizzando i pacchetti di Github che sono gratuiti per tutti i progetti pubblici. 
Per farlo basta aggiungere l'opportuna action e dovrebbe andare.
Comunque è un'ottima idea e sarebbe perfettamente in linea con tutti gli altri progetti. Appena ho un attimo di tempo creo la action e magari aggiungiamo il docker compose nel README.